sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -75,10 +75,9 @@ target_include_directories(
       $<INSTALL_INTERFACE:include>
 )
 
-# libxml2 headers and libraries will be changed to PRIVATE in the future.
-target_include_directories(azure-storage-common PUBLIC ${LIBXML2_INCLUDE_DIRS})
 target_link_libraries(azure-storage-common PUBLIC Azure::azure-core)
-target_link_libraries(azure-storage-common INTERFACE Threads::Threads ${LIBXML2_LIBRARIES})
```

Comment:
What happened to `Threads::Threads`? How come it is no longer needed?

Reply:
`Azure::azure-core` depends on `Threads::Threads`. It will transitively populate dependency libraries of `azure-storage-common`